datafusion/expr/src/planner.rs

```diff
@@ -167,14 +170,14 @@ pub trait ExprPlanner: Debug + Send + Sync {
 
     /// Plan an extract expression, such as`EXTRACT(month FROM foo)`
     ///
-    /// Returns origin expression arguments if not possible
+    /// Returns original expression arguments if not possible
```

Comment:
This is a good change. nit Could go in separate PR to keep PR size lower.

Reply:
I think we can keep this